lib/aposResponse.js

```diff
@@ -23,6 +23,14 @@ function looksLikeChunkedEncoding(buffer) {
 
 export default async function aposResponse(req) {
   try {
+    // Host header should not contain the protocol or a path
+    if (req.headers.get('host').includes('/')) {
+      return new Response('Invalid Host header', {
+        status: 400,
+        statusText: 'Bad Request'
+      });
+    }
```

Comment:
After I patched the things, I sanity checked this and it works as expected. I had to add `localhost/` to `server.allowedHosts` array in the astro config in order to test it (because Vite just refued to deliver it).

Reply:
yes exactly, I should have documented this!

<img width="348" height="167" alt="image" src="https://github.com/user-attachments/assets/f4b5620e-9ca8-4f83-82fd-a8056c9a1fbb" />